docs/protocol-p2p-network/orphan-blocks.md

```diff
@@ -2,15 +2,15 @@
 
 ---
 
-Blocks-first nodes may download orphan blocks---blocks whose previous
+dcrd may download orphan blocks---blocks whose previous
 block header hash field refers to a block header this node
 hasn't seen yet. In other words, orphan blocks have no known parent
 (unlike stale blocks, which have known parents but which aren't part of
 the best block chain).
 
-![Difference Between Orphan And Stale Blocks](/img/dev/en-orphan-stale-definition.svg?)
+![Difference Between Orphan And Stale Blocks](/img/protocol-p2p-network/en-orphan-stale-definition.svg)
 
-When a blocks-first node downloads an orphan block, it will not validate
+When dcrd downloads an orphan block, it will not validate
 it. Instead, it will send a `getblocks` message to the node which sent
 the orphan block; the broadcasting node will respond with an `inv` message
```

Comment:
The ';'s are tripping me up here. If I read them as commas, it works. Is this a legal use of `;`s?